src/schemas/json/project.json

```diff
@@ -655,6 +655,10 @@
 		"runtimes": {
 			"description": "List of runtime identifiers supported by this project (used when building standalone applications).",
 			"type": "object"
+		},
+		"userSecretsId": {
+			"description": "Specify a unique id to manage the development time user secrets of the application.",
```

Comment:
- The identifier doesn't "manage" anything. What it actually does is link the project to a specific set of secrets.
- On the other hand, I like emphasizing the uniqueness of the identifier and that this is about development-time.
- Minor but why say "of the application"?
- Suggest "id" -> "identifier".